plugins/woocommerce/includes/admin/class-wc-admin-post-types.php

```diff
@@ -200,7 +200,7 @@ public function order_updated_messages( array $messages ) {
 			9  => sprintf(
 			/* translators: %s: date */
 				__( 'Order scheduled for: %s.', 'woocommerce' ),
-				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), strtotime( $theorder->get_date_created() ) ) . '</strong>'
+				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), $theorder->get_id() !== 0 ? strtotime( $theorder->get_date_created() ) : false ) . '</strong>'
```

Comment:
This doesn't solve the issue (the warning is back). We should check that `$theorder->get_date_created()` is not null, not that `$theorder->get_id() != 0` (which shouldn't be the case ever if everything's correct).

Please take a look at the initial suggestion I gave, which was just a shortcut to this check. Also, passing `false` to `strtotime()`, while it doesn't produce an error, isn't great either. We do have the correct time, either from the order object itself or from `$post`.